dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -330,7 +330,7 @@ public enum Status {
 
     //plugin
     PLUGIN_NOT_A_UI_COMPONENT(110001, "query plugin error, this plugin has no UI component", "查询插件错误，此插件无UI组件"),
-    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query plugins result is null", "查询插件为空"),
+    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query alarm plugins result is empty, please check whether the alarm component is started and the plugin is successfully registered", "查询告警插件为空, 请检查告警组件是否启动相关告警插件是否注册成功"),
```

Comment:
Is it better to use ```请检查告警插件是否启动成功以及相关告警插件是否注册成功```?

Reply:
yes, I optimized it according to your suggestion

Reply:
I think the document has also been updated, I will modify the relevant content of the document later